metaStoreApp/spec/controllers/medias_controller_spec.rb

```diff
@@ -2,54 +2,59 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::MediasController, type: :controller do
-  before(:context) do
+  before(:each) do
     @account = FactoryBot.create(:account)
-  end
-
-  after(:context) do
-    @account.destroy
+    for i in 60..65 do
```

Comment:
there is an rspec/factorybot way of creating multiple entries

Reply:
okay. I will check that

Reply:
I can create multiple entries with same value but asset_id is primary key so it raises error.
If i create multiple entries with different values then its changing dynamically (media.sequence(:title) { |n| "title#{n}" }) and i cannot check in rspec